src/Cake.AndroidAppManifest_.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@gentledepp since the nuget package info is in the .csproj, any reason why you added this .nuspec here?